nextflow_schema.json

```diff
@@ -706,6 +706,148 @@
+                },
+                "damage_manipulation_rescale_seqlength": {
+                    "type": "integer",
+                    "default": 12,
+                    "description": "Length of read sequence to use from each side for rescaling. Can be overridden by `--rescalelength*p`.",
+                    "help_text": "Specify the length from the end of the read that mapDamage should rescale at both ends.\n\n> Modifies the `--seq-length` parameter of mapDamage2.",
+                    "fa_icon": "fas fa-ruler-horizontal"
+                },
+                "damage_manipulation_rescale_length_5p": {
+                    "type": "integer",
+                    "default": 0,
+                    "description": "Length of read for mapDamage2 to rescale from 5p end. Only used if not 0 otherwise `--rescale_seqlength` used.",
+                    "help_text": "Specify the length from the end of the read that mapDamage should rescale. Overrides `--rescale_seqlength`.\n\n> Modifies the `--rescale-length-5p` parameter of mapDamage2.",
+                    "fa_icon": "fas fa-balance-scale-right"
+                },
+                "damage_manipulation_rescale_length_3p": {
+                    "type": "integer",
+                    "default": 0,
+                    "description": "Length of read for mapDamage2 to rescale from 3p end. Only used if not 0 otherwise `--rescale_seqlength` used.",
+                    "help_text": "Specify the length from the end of the read that mapDamage should rescale. Overrides `--rescale_seqlength`.\n\n> Modifies the `--rescale-length-3p` parameter of mapDamage2.",
+                    "fa_icon": "fas fa-balance-scale-left"
+                },
+                "run_pmd_filtering": {
+                    "type": "boolean",
+                    "description": "Turn on PMDtools filtering.",
+                    "help_text": "Specifies to run PMDTools for damage based read filtering in sequencing libraries. Turned off by default.",
+                    "fa_icon": "fas fa-power-off"
+                },
+                "damage_manipulation_pmdtools_threshold": {
+                    "type": "integer",
+                    "default": 3,
+                    "fa_icon": "far fa-chart-bar",
+                    "description": "Specify PMDScore threshold for PMDTools.",
+                    "help_text": "Specifies the PMDScore threshold to use in the pipeline when filtering BAM files for DNA damage. Only reads which surpass this damage score are considered for downstream DNA analysis. By default set to `3` if not set specifically by the user.\n\n> Modifies PMDTools parameter: `--threshold`"
```

Comment:
Don't need to mention defeualt, schema will display this for you with the `default` field